pyanaconda/ui/cockpit/anaconda-webui/Makefile.am

```diff
@@ -57,6 +58,13 @@ bots:
 	if [ -n "$$COCKPIT_BOTS_REF" ]; then git -C bots fetch --quiet --depth=1 origin "$$COCKPIT_BOTS_REF"; git -C bots checkout --quiet FETCH_HEAD; fi
 	@echo "checked out bots/ ref $$(git -C bots rev-parse HEAD)"
 
+# FIXME: we need to download this from somewhere if not existing
+$(ISO_FILE):
+	test ! -e $(ISO_FILE) && curl https://download.fedoraproject.org/pub/fedora/linux/development/rawhide/Server/x86_64/os/images/boot.iso --output $(ISO_FILE)
```

Comment:
Leftover from previous round: You really don't need that `test` call. That's what make rules are for.